metadata-io/src/main/java/com/linkedin/metadata/graph/elastic/GraphFilterUtils.java

```diff
@@ -158,7 +161,7 @@ public static QueryBuilder getEdgeTimeFilterQuery(
    */
   private static QueryBuilder buildTimeWindowFilter(
       final long startTimeMillis, final long endTimeMillis) {
-    final BoolQueryBuilder timeWindowQuery = QueryBuilders.boolQuery();
+    final BoolQueryBuilder timeWindowQuery = QueryBuilders.boolQuery().minimumShouldMatch(1);
```

Comment:
Might be helpful from a clarity perspective to colocate this with the should calls below, seems odd to have that pattern for the other two, but not this one.

Reply:
That's because we always populate the should, no potential cases where it might be empty.